package.json

```diff
@@ -5,13 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "server": "nodemon server.js"
+    "server": "nodemon index.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/saltytrolls-bw-may-19/BackEnd.git"
   },
-  "author": "Ajoe Bacus",
+  "author": "Aljoe Bacus",
```

Comment:
I spelled your name wrong? I'm such a punk...

Reply:
Hahaha it's fine, pretty sure it was just a forgotten key press.